app/views/pages/core_materials_info.html.erb

```diff
@@ -181,7 +186,8 @@
   <li>reflect on what they’ve learnt</li>
 </ul>
 
-<p><%= govuk_link_to "See an example of a UCL topic", "https://www.early-career-framework.education.gov.uk/ucl/ucl/2-understanding-teachers-as-role-models/4-making-productive-use-of-assessment/5-planning-effective-and-manageable-marking-and-feedback/" %></p>
+<p><%= govuk_link_to "See an example of a UCL topic",
+                     "https://www.early-career-framework.education.gov.uk/ucl/ucl/2-understanding-teachers-as-role-models/4-making-productive-use-of-assessment/5-planning-effective-and-manageable-marking-and-feedback/" %></p>
```

Comment:
Missed one?

Reply:
I wish :P 

This points at a page rather than at a pdf. We will need to replace it differently. Somehow.